sys/include/net/gnrc/ipv6/nib/conf.h

```diff
@@ -125,12 +125,19 @@ extern "C" {
 #endif
 
 /**
- * @brief    activate router advertising at interface start-up
+ * @brief   activate router advertising at interface start-up
  */
 #ifndef CONFIG_GNRC_IPV6_NIB_ADV_ROUTER
 #define CONFIG_GNRC_IPV6_NIB_ADV_ROUTER               0
 #endif
 
+/**
+ * @brief   activate authoritative border router functionality at interface start-up
+ */
+#ifndef CONFIG_GNRC_IPV6_NIB_ABR
+#define CONFIG_GNRC_IPV6_NIB_ABR                    CONFIG_GNRC_IPV6_NIB_6LBR
```

Comment:
Maybe you can add a line to the docs of the border router (or border router example?) about how to possibility to make a 6lbr non-authorative.